src/DbCreateCommand.php

```diff
@@ -9,7 +9,7 @@
  *
  * @package App\Console\Commands
  */
-class DbCreate extends Command
+class DbCreateCommand extends Command
```

Comment:
There is no need to use "Command" as postfix. See https://laravel.com/docs/5.8/artisan#command-structure

Reply:
I'm fine with this. ;)